.gitlab/pipelines/stages/deploy/setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1090
+source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
+
+wait_for_file() {
+  local file_path="$1"
+  local timeout="${2:-30}"
+  until [[ -f "${file_path}" ]] || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
+  if [[ "${timeout}" == 0 ]]; then return 1; fi
+  return 0
+}
+
+printf "k3s: %s\n" "$(bazel run @k3s//executable -- --version 2> /dev/null)"
+sudo -E "$(command -v bazel)" run @k3s//executable -- server 1> /tmp/k3s.out.log 2> /tmp/k3s.err.log &
```

Comment:
I'll dump the logs to /dev/null because we don't really need them on the day-to-day run. At this point, we assume it's working as expected. If it stops working, then it's another story and we can dump the logs to a file and investigate.